packages/blocks/table/src/devRenderer.tsx

```diff
@@ -2,187 +2,24 @@
  * webpack-dev-server entry point for debugging.
  * This file is not bundled with the library during the build process.
  */
-import React, { useCallback, useMemo, useState } from "react";
+import React from "react";
 import ReactDOM from "react-dom";
 import { tw } from "twind";
 
-import {
```

Comment:
Once refactored for the correct links approach we can replace all this with `MockBlockDock`

Reply:
Good flag. Makes sense